### PR TITLE
Fix additional typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found additional misspellings with `typos`.

### CODE REVIEW

**Comment by szepeviktor** [comment]
Please consider replacing codespell with typos.

`.typos.toml`

```toml
[files]
extend-exclude = [
    ".github/workflows/static_checks.yml",
    ".mailmap",
    "AUTHORS.md",
    "DONORS.md",
    "core/input/gamecontrollerdb.txt",
    "core/string/locales.h",
    "doc/translations/",
    "editor/translations/",
    "misc/dist/linux/org.godotengine.Godot.desktop",
    "misc/scripts/codespell.sh",
    "thirdparty/",
]
ignore-hidden = false

[default]
extend-ignore-re = [
    "[abc]CROSS[abc]p",
    "_USCALED",
    "_Uscaled",
    "Big Sur",
    "FO_DELETE",
    "GDSCript",
    "HB_SCRIPT_VAI",
    'Prints "Hel"',
    'U"Vai"',
]

[default.extend-words]
"alph" = "alph"
"ba" = "ba"
"clen" = "clen"
"findn" = "findn"
"inout" = "inout"
"lod" = "lod"
"mis" = "mis"
"nd" = "nd"
"numer" = "numer"
"ot" = "ot"
"requestor" = "requestor"
"tpos" = "tpos"
```

File: doc/classes/AnimationNode.xml [hunk 1]
```diff
@@ -80,12 +80,12 @@
 			<param index="0" name="animation" type="StringName" />
 			<param index="1" name="time" type="float" />
 			<param index="2" name="delta" type="float" />
-			<param index="3" name="seeked" type="bool" />
```
Comment (AThousandShips) [inline doc/classes/AnimationNode.xml:83]: This is not correct, `seeked` is a term used across the codebase

Comment (szepeviktor) [reply doc/classes/AnimationNode.xml:83]: All right. Reverting ...

**Review by AThousandShips — CHANGES_REQUESTED** [review]
None of the cases renaming methods, properties, or constants can be done, they are compatibility breaking changes

File: doc/classes/ResourceUID.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 		A singleton that manages the unique identifiers of all resources within a project.
 	</brief_description>
 	<description>
-		Resource UIDs (Unique IDentifiers) allow the engine to keep references between resources intact, even if files can renamed or moved. They can be accessed with [code]uid://[/code].
+		Resource UIDs (Unique IIdentifiers) allow the engine to keep references between resources intact, even if files can renamed or moved. They can be accessed with [code]uid://[/code].
```
Comment (aaronfranke) [inline doc/classes/ResourceUID.xml:7]: This is wrong.

File: doc/classes/Projection.xml [hunk 3]
```diff
@@ -246,7 +246,7 @@
 				Returns [code]true[/code] if this [Projection] performs an orthogonal projection.
 			</description>
 		</method>
-		<method name="jitter_offseted" qualifiers="const">
+		<method name="jitter_offsetted" qualifiers="const">
```
Comment (aaronfranke) [inline doc/classes/Projection.xml:249]: This breaks compatibility, so it can't be done in a stable release.

**Comment by szepeviktor** [comment]
I think this is above my paygrade.
You can run `typos` with the above configuration and fiddle with the results.
I am sorry.

**Comment by AThousandShips** [comment]
Thank you for your contribution, but it's important to consider things like compatibility, and internal lingo, or programmer terminology

**Comment by AThousandShips** [comment]
Activating `typos` would force changes to already existing typos when referencing methods etc., which would cause problems

**Comment by szepeviktor** [comment]
Glad to contribute.

File: doc/classes/AnimationNode.xml (hunk 1 above)
Comment (Zireael07) [reply doc/classes/AnimationNode.xml:83]: The fact it's used across codebase doesn't change the fact that it's not proper, grammatical English and typo-catchers WILL point this out

Comment (AThousandShips) [reply doc/classes/AnimationNode.xml:83]: Exactly why sweeping typo fixes isn't a useful tool for such

Further it isn't necessarily un-grammatical, "seek" in this case isn't the same as "to seek" necessarily, take "computer mouse" , "computer mouses" for example

Comment (Zireael07) [reply doc/classes/AnimationNode.xml:83]: If it isn't "to seek" it should be noted somewhere (e.g. where it's first used) as it's bound to confuse both non-native English speakers delving into Godot code AND typo catchers

Comment (AThousandShips) [reply doc/classes/AnimationNode.xml:83]: In fact this is established computer terminology:
https://en.wiktionary.org/wiki/seeked

For over 30 years now

Comment (Zireael07) [reply doc/classes/AnimationNode.xml:83]: Wow, TIL! Link worth putting in a comment somewhere so that other non-natives don't get confused

Comment (AThousandShips) [reply doc/classes/AnimationNode.xml:83]: I don't think that would help much, you would have to find that one specifically, and you couldn't do it to the docs

**Pull request closed without merging**